python/fledge/services/core/scheduler/scheduler.py

```diff
@@ -1287,6 +1305,9 @@ async def disable_schedule(self, schedule_id: uuid.UUID, bypass_check=None):
                 if task_future.cancel() is True:
                     await self._wait_for_task_completion(task_process)
 
+        # Delete startup token
+        self._startupTokens.pop(schedule.name, None)
```

Comment:
on succesful registration, the set/dict holding that single use token should pop that.